# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: .cargo/config.toml.in
# Commit: b3109ad14685
# Full Hash: b3109ad14685624ada7861083ff5332aec9a7818
# Author: Andy Leiserson <aleiserson@mozilla.com>
# Date: 2025-10-07 17:00 +0000
# Description:
#   Bug 1992842 - Update wgpu to upstream 1072b878 (2025-10-06) r=webgpu-reviewers,supply-chain-reviewers,ErichDonGubler
# ==============================================================================

diff -r 1d3f000cf1f9 -r b3109ad14685 .cargo/config.toml.in
--- a/.cargo/config.toml.in	Tue Oct 07 16:51:44 2025 +0000
+++ b/.cargo/config.toml.in	Tue Oct 07 17:00:40 2025 +0000
@@ -40,9 +40,9 @@
 rev = "43c22248d136c8b38fe42ea709d08da6355cf04b"
 replace-with = "vendored-sources"
 
-[source."git+https://github.com/gfx-rs/wgpu?rev=8c4aebc0c1b75e3f7412dfc180745a0717fc30ec"]
+[source."git+https://github.com/gfx-rs/wgpu?rev=1072b87894b7c3084f80bc9d2679e822435186ba"]
 git = "https://github.com/gfx-rs/wgpu"
-rev = "8c4aebc0c1b75e3f7412dfc180745a0717fc30ec"
+rev = "1072b87894b7c3084f80bc9d2679e822435186ba"
 replace-with = "vendored-sources"
 
 [source."git+https://github.com/glandium/allocator-api2?rev=ad5f3d56a5a4519eff52af4ff85293431466ef5c"]